capella2polarion/converters/element_converter.py

```diff
@@ -57,6 +59,42 @@ def _format(texts: list[str]) -> dict[str, str]:
     return requirement_types
 
 
+def _resolve_capella_attribute(
+    converter_data: data_session.ConverterData, attribute: dict[str, t.Any]
+) -> polarion_api.TextContent | str:
+    def _handle_value(
+        value: str | enum.Enum, capella_attr: str, enum_id: str | None
+    ) -> polarion_api.TextContent | str:
+        if enum_id is not None:
+            return polarion_api.TextContent(
+                type=f"enum:{enum_id}",
+                value=value if isinstance(value, str) else value.name,
+            )
```

Comment:
Did you check, if this is the right way to send enums via the REST API?

Reply:
Yes. Indeed the REST API just accepts the value.